types/bpmn-moddle/index.d.ts

```diff
@@ -1,31 +1,23 @@
-// Type definitions for bpmn-moddle 5.1
+// Type definitions for bpmn-moddle 8.0
 // Project: https://github.com/bpmn-io/bpmn-moddle
 // Definitions by: DefinitelyTyped <https://github.com/DefinitelyTyped>
+//                 Artur Avetisov <https://github.com/Droid0012>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 5.2
 
 declare namespace BPMNModdle {
-    type AdHocOrdering = "Parallel" | "Sequential";
-    type AssociationDirection = "None" | "One" | "Both";
-    type ChoreographyLoopType =
-        | "None"
-        | "Standard"
-        | "MultiInstanceSequential"
-        | "MultiInstanceParallel";
-    type EventBasedGatewayType = "Parallel" | "Exclusive";
-    type GatewayDirection =
-        | "Unspecified"
-        | "Converging"
-        | "Diverging"
-        | "Mixed";
-    type ItemKind = "Physical" | "Information";
-    type MultiInstanceBehaviour = "None" | "One" | "All" | "Complex";
-    type ProcessType = "None" | "Public" | "Private";
-    type RelationshipDirection = "None" | "Forward" | "Backward" | "Both";
+    type AdHocOrdering = 'Parallel' | 'Sequential';
```

Comment:
Most of this PR is just formatting changes. That's making it very hard to review.